.github/workflows/lint.yml

```diff
@@ -29,3 +29,25 @@ jobs:
           skip-cache: true
           version: latest
           args: --timeout=30m ./...
+  nilaway:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "stable"
+      - name: Go mod cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Install dependencies
+        run: go mod tidy
+      - name: Install Nilaway
+        run: go install go.uber.org/nilaway/cmd/nilaway@latest
+      - name: Run Nilaway
+        run: nilaway ./...
```

Comment:
I'm not sure if it's correct, did you test this?

Reply:
> I'm not sure if it's correct, did you test this?

Yes, already tested.